plugin.xml

```diff
@@ -1,7 +1,7 @@
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
 xmlns:android="http://schemas.android.com/apk/res/android"
            id="cordova-plugin-firebase-dynamiclinks2"
-      version="4.0.8">
+      version="4.0.9">
```

Comment:
Need to bump the version in package.json too

Reply:
Updated the version in package.json